sample.ini

```diff
@@ -1,29 +1,37 @@
 [General]
-
-hostname = 0.0.0.0
 
-portNumber = 8000
+hostname = 0.0.0.0
 
-enableTracing = 0
+portNumber = 8000
+
+enableTracing = 0
 
 ; If any symbols of interest aren't available locally (e.g. Windows DLLs), uncomment line below
-
+
 ; remoteSymbolServer = http://symbolapi.mozilla.org:80/
 
-
-
-; These options are unlikely to need editing by users
 
-maxCacheEntries = 2000000
 
-mruSymbolStateFile = /tmp/snappy-mru-symbols.json
+; These options are unlikely to need editing by users
+
+maxCacheEntries = 2000000
+
+mruSymbolStateFile = /var/snappy-mru-symbols.json
 maxMRUSymbolsPersist = 10
 
-[SymbolPaths]
+[Log]
+
+maxFiles = 10
+
+maxFileSize = 1048576
+
+logPath = /tmp/log/snappy
+
+[SymbolPaths]
 
-Firefox = /mnt/netapp/breakpad/symbols_ffx
+Firefox = /mnt/netapp/breakpad/symbols_ffx
```

Comment:
i'd like to change how these symbol paths are handled.. the official symbolapi.mozilla.org server won't have access to any symbol directories when doing symbol lookups, so it doesn't make sense to check for symbols on-disk in production before checking the (future) disk cache or fetching from S3. Windows devs will need to use on-disk symbols + remoteSymbolServer only. So using S3 or local symbols should be mutually exclusive options